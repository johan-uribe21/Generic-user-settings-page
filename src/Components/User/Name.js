import React from 'react';

// stateless display component
const Name = ({name}) => {

  return (
    <div className="beam-text f3 flex justify-center">
      <p>Name: {name}</p>
    </div>
  )
}

export default Name;