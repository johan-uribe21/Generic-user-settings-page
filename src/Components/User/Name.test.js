import {shallow, configure} from 'enzyme';
import Name from './Name';
import React from 'react';
import Adapter from 'enzyme-adapter-react-16';

configure({ adapter: new Adapter() }); // need this line to use enzyme

it('expect to render name correctly', () => {
  expect(shallow(<Name/>)).toMatchSnapshot();
});
