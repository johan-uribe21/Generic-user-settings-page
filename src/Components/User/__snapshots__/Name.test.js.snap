// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render name correctly 1`] = `ShallowWrapper {}`;
